embedded-graphics/src/primitives/mod.rs

```diff
@@ -86,15 +86,15 @@ pub trait Primitive: Dimensions {
 /// ```
 #[macro_export]
 macro_rules! egcircle {
-    (center = $center:expr, radius = $r:expr $(,)?) => {{
+    (top_left = $top_left:expr, diameter = $d:expr $(,)?) => {{
```

Comment:
Additional variants for `egcircle!(center = ..., diameter = ..., ...)` would also be useful.

Reply:
Yes, good idea. Done.